release-content/0.14/release-notes/13417_implement_the_full_set_of_sort_methods_on_QueryIter.md

```diff
@@ -1 +1,114 @@
-TODO
+Currently, query iteration order is not guaranteed. If we wish to work with our query items in a certain order, we need sorting.
+We might want sort our queries to f.e. display a list of things, or fold over our query with some order-dependent math operation.
```

Comment:
```suggestion
We might want to display the scores of the players in order, or ensure a consistent iteration order for the sake of networking stability.
```

f.e. isn't a particularly common abbreviation in English, and I've made the examples more concrete / game-relevant :)

Reply:
Done!